packages/core/src/component.ts

```diff
@@ -14,6 +14,15 @@ export interface ProviderMap {
   [key: string]: Constructor<Provider<BoundValue>>;
 }
 
+/**
+ * Static assets settings
+ */
+export type staticAssetsSettings = {
```

Comment:
`export interface StaticAssetsSettings` (use interface, not type; interface names start with a capital letter)

I would also pick a slightly different name, e.g. `staticAssetsDefinition` or `staticAssetsConfig`.